### ISSUE TITLE
REFPROP and mixtures problems

### ISSUE THREAD

**Issue by ibell** [ibell/coolprop#28]
When using refprop mixtures I face an issue with wrong enthalpies and other values.
Predefined mixtures as pseudo pure liquids CP/RP work beautifully out of the box with my diagram generator, see attached file.

here an example:
I wanted to plot a p-h Diagram and get the Enthalpies for the start of the
saturation curves, left bottom and right bottom. They differ from Refprop values!
my boudaries are 30 to 5000 kPa, 100 to 640 kJ/kg

Definition from Refprop 9.1 R407F.MIX :

R407F='REFPROP-MIX:R32[0.473194694453358]&R125[0.205109095413331]&R134a[0.321696210133311]'

hlb=cp.Props('H','P',30,'Q',0,R407F) # 30 kPa saturated liquid
hrb=cp.Props('H','P',30,'Q',1,R407F) # 30 kPa saturated vapour
h200=cp.Props('H','T',273.15,'Q',0,R407F) # saturated liquid at 0°C IIR

print(hlb,hrb,h200)
output >
215.5452457491837 381.9412090376607 224.3523908950257
should be
107.34 392.47 200.00

reversely it fails too:
print(cp.Props('T','H',107.34,'P',30,R407F)-273.15)
print(cp.Props('T','H',392.47,'P',30,R407F)-273.15)
output:
-68.94365355255812
-46.71547963272596
should be:
-68.13
-61.4 ... in no case above -60°C

**Comment by ibell** [comment]
I have verified the problem exists, though I get different values.  I thought originally it was just a reference state problem, but that doesn't seem to be the case.

**Comment by jowr** [comment]
No, neither is it connected to mass fraction / mole fraction conversion. The values I get are:

```
Refprop:  101.417454409 408.803313633 201.271436587
CoolProp: 197.940985930 381.941209038 224.352390895
```

for hlb, hrb and h200...

**Comment by ibell** [comment]
Using C++ from master I get:

```
char R407F[]="REFPROP-MIX:R32[0.473194694453358]&R125[0.205109095413331]&R134a[0.321696210133311]";
double hL = Props("H",'T',300,'Q',0,R407F);
double hV = Props("H",'T',300,'Q',1,R407F);
double h200 = Props("H",'T',273.15,'Q',0,R407F);
```

256.29664842979935 425.13997290037872 224.35239089502571

There is clearly something odd going on. ( I fixed temperature rather than pressure, but anyway)

@jowr, are you using REFPROP 9.1 or 9.0?

**Comment by ibell** [comment]
We should all check temperature and densities as well for liquid and vapor to see if those come out the same.  Enthalpies are trickier due to the reference state issues

**Comment by jowr** [comment]
I am not sure. I think I use 9.1

**Comment by ibell** [comment]
Seems like a bug in REFPROP related to the handling of the reference state since the densities are spot on for all three inputs between CoolProp+REFPROP and REFPROP GUI.

**Comment by ibell** [comment]
Fixed in revision 0518d962a269fc83520ba14434f43e82be7c2406
